rsts/deployment/plugins/index.rst

```diff
@@ -57,3 +66,4 @@ to use these plugins in your workflows written in ``flytekit``.
     aws/index
     gcp/index
     webapi/index
+    memverge/index
```

Comment:
We should update [agents/index.rst](https://github.com/flyteorg/flyte/blob/master/rsts/deployment/agents/index.rst) instead.

Reply:
updated